cmd/cli/root.go

```diff
@@ -96,6 +97,13 @@ func NewRootCommand(opt *option.Option, rs *rootState) (*cobra.Command, error) {
 	cmd.AddCommand(login.NewCommand(opt))
 	cmd.AddCommand(stage.NewCommand(opt))
 	cmd.AddCommand(newVersionCommand(opt))
+	cmd.AddCommand(
+		cobracompletefig.CreateCompletionSpecCommand(
+			cobracompletefig.Opts{
+				Use: "fig",
+			},
+		),
+	)
```

Comment:
This subcommand is hidden. It's run during the release process to generate spec.